examples/clusterDecisionResource/README.md

```diff
@@ -0,0 +1,57 @@
+# How the duck type generator works for clusterDecisionResource
+1. The Duck type generator reads a configurable status format:
```

Comment:
References to 'Duck type generator' here can presumably now be Cluster Decision Resource generator.